torchvision/datasets/video_utils.py

```diff
@@ -215,12 +227,12 @@ def compute_clips_for_video(video_pts, num_frames, step, fps, frame_rate):
             warnings.warn("There aren't enough frames in the current video to get a clip for the given clip length and "
                           "frames between clips. The video (and potentially others) will be skipped.")
         if isinstance(idxs, slice):
-            idxs = [idxs] * len(clips)
+            idxs = [idxs] * len(clips)  # type: ignore[assignment]
         else:
             idxs = unfold(idxs, num_frames, step)
-        return clips, idxs
+        return clips, idxs  # type: ignore[return-value]
```

Comment:
Why do we need to ignore here? Doesn't that mean that the return annotation is wrong?